extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -1217,6 +1256,21 @@ export class AmpLightboxGallery extends AMP.BaseElement {
     });
   }
 
+  // TODO: are there utilities for this? if not where should they go?
+  to2Digits_(i) {
+    return i < 10 ? '0' + i : i;
```

Comment:
I felt like we had a function that does this, but I can't remember. I would add it to `string.js` as a utility e.g. leftPad